pyproject.toml

```diff
@@ -120,7 +120,7 @@ asyncio_mode = "auto"
 asyncio_default_fixture_loop_scope="function"
 
 [tool.coverage.report]
-omit = ["param/version.py"]
+omit = ["param/version.py", "tests/testimports.py"]
```

Comment:
This belongs to the other PR, but I don't think moving it over to it is important.